polar2grid/readers/mersi2_l1b.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+import logging
+from polar2grid.readers import ReaderWrapper, main
+
+LOG = logging.getLogger(__name__)
+
+
+class Frontend(ReaderWrapper):
+    FILE_EXTENSIONS = ['.h5']
```

Comment:
This file extension has to match what the actual files will be. For mersi2 this is ".HDF". See https://github.com/pytroll/satpy/blob/master/satpy/etc/readers/mersi2_l1b.yaml

Please double check the other readers too.